subprojects/core/src/integTest/groovy/org/gradle/api/ApplyPluginIntegSpec.groovy

```diff
@@ -39,7 +38,8 @@ class ApplyPluginIntegSpec extends AbstractIntegrationSpec {
     }
 
     @Issue("GRADLE-2358")
-    @FailsWith(UnexpectedBuildFailure) // Test is currently failing
+    @FailsWith(UnexpectedBuildFailure)
+    // Test is currently failing
```

Comment:
Maybe your auto-formatter change that. Let's revert that to have a smaller footprint of changes. Makes reviewing easier.